site/en/guide/dtensor_overview.ipynb

```diff
@@ -107,9 +109,9 @@
         "id": "O3pG29uZIWYO"
       },
       "source": [
-        "Once installed, import `tensorflow` and `tf.experimental.dtensor`. Then configure TensorFlow to use 6 virtual CPUs.\n",
```

Comment:
@MarkDaoust full API with `tf.experimental`  is mentioned in the beginning of the section, so shortened this.

Later, "vCPU" -> "virtual CPU"